### PR TITLE
Improve handling of ssl profile

### PR DESCRIPTION
**Pull request by grs**
This should make the pkg/site/routeraccess .go (moved from pkg/kube/site/routeraccess.go) and pkg/site/link .go usable for other platforms where different paths may be required.

### CODE REVIEW

File: pkg/site/routeraccess.go [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+		return nil
+	}
+	var connectors []qdr.Connector
+	if role, ra := m.findInterRouterRole(); role != nil {
+		for _, group := range targetGroups {
+			name := group
+			connector := qdr.Connector{
+				Name:       name,
+				Host:       group,
+				Role:       qdr.RoleInterRouter,
```
Comment (fgiorgetti) [inline pkg/site/routeraccess.go:42]: Is this for the HA?

**Review by fgiorgetti — COMMENTED** [review]
Overall it looks good to me.

File: pkg/kube/site/site.go [hunk 2]
```diff
@@ -31,13 +30,11 @@ type Site struct {
...
 	name        string
 	namespace   string
-	siteId      string
-	config      *types.SiteConfigSpec
 	controller  *kube.Controller
 	bindings    *site.Bindings
 	links       map[string]*site.Link
 	errors      map[string]string
-	linkAccess  LinkAccessMap
+	linkAccess  site.RouterAccessMap
```
Comment (fgiorgetti) [inline pkg/kube/site/site.go:37]: Maybe rename it to routerAccess ?

File: api/types/crds/skupper_link_crd.yaml [hunk 3]
```diff
@@ -15,55 +15,39 @@ spec:
...
+                  items:
+                    type: object
+                    properties:
+                      name:
+                        type: string
+                      host:
+                        type: string
+                      port:
+                        type: string
+                      group:
```
[the remark this answers is not in the document]
Comment (grs) [reply api/types/crds/skupper_link_crd.yaml:29]: It is used for HA to group together the ports for a given router

File: pkg/site/routeraccess.go (hunk 1 above)
Comment (grs) [reply pkg/site/routeraccess.go:42]: yes

File: pkg/kube/site/site.go (hunk 2 above)
Comment (grs) [reply pkg/kube/site/site.go:37]: The reason it is called linkAccess is that is only controlling the access for linking.  A separate field will control management access.

**Pull request closed without merging**